src/commands/Management/permissionNodes.ts

```diff
@@ -21,12 +24,19 @@ export class UserCommand extends SkyraCommand {
 	public async add(message: GuildMessage, args: SkyraCommand.Args) {
 		const target = await args.pick('roleName').catch(() => args.pick('member'));
 		const action = await args.pick(UserCommand.type);
-		const command = await args.pick('command');
 
-		if (!this.checkPermissions(message, target)) this.error(LanguageKeys.Commands.Management.PermissionNodesHigher);
+		// Permission Nodes do not allow allows for the @everyone role:
+		if (target.id === message.guild.id && action === PermissionNodeAction.Allow) {
+			this.error(LanguageKeys.Commands.Management.PermissionNodesCannotAllowEveryone);
+		}
```

Comment:
Hate to do it to ya.. but this needs a DB migration to remove any existing assignments to the `@everyone` role, and remove those pnodes.